tests/flow/test_entity_update.py

```diff
@@ -134,3 +139,29 @@ def test13_invalid_complex_type_in_array(self):
                 self.env.assertTrue(False)
             except ResponseError as e:
                 self.env.assertContains("Property values can only be of primitive types or arrays of primitive types", str(e))
+
```

Comment:
I'm missing a few tests:
1. multiple updates to the same entity, e.g. `MATCH (x), (n) WHERE ID(n) = 0 SET n = x`
2. multiple updates to the same entity, e.g. `MATCH (x), (n) WHERE ID(n) = 0 SET n += x`
3. multiple updates to the same entity, e.g. `MATCH (x), (n) WHERE ID(n) = 0 SET n=x, n += x`

I would like to see tests where an entity goes through a series of changes: updating existing attributes, introducing new ones and deleting existing attributes, at the end we need to make sure all relavent indices been updated properly.